src/lib/analytics/amplitude.ts

```diff
@@ -22,7 +22,7 @@ export const identify = (property: AnalyticsUserProperty) => {
     detail: { property: propertyTypeToLog, propertyValue: property.payload },
   });
   dd.setContextProperty(propertyTypeToLog, property.payload);
-  dd.log(`set context item: ${propertyTypeToLog}`, dd.getContext());
+  dd.info(`set context item: ${propertyTypeToLog}`, dd.getContext());
```

Comment:
Does it make sense to just add `dd.info` to `track`? or was it decided that is too much events

Reply:
this was discussed [in slack](https://dydx-team.slack.com/archives/C042V24U87Q/p1724109154414829?thread_ts=1724101364.841789&cid=C042V24U87Q) but we decided to be very conservative about the # of events we send and track the volume first